source/docs/software/pathplanning/pathweaver/creating-pathweaver-project.rst

```diff
@@ -34,9 +34,9 @@ To create a PathWeaver project, click on "Create project" and then fill out the
 
 **Export Unit:** The units to be used when exporting the paths and waypoints. If you are planning to use WPILib Trajectories, then you should choose :guilabel:`Always Meters`. Choosing :guilabel:`Same as Project` will export in the same units as :guilabel:`Length Unit` above.
 
-**Max Velocity:** The max speed of the robot for trajectory tracking. This does not need to be the maximum attainable speed of the robot, but just the max speed that you want to run trajectories at.
+**Max Velocity:** The max speed of the robot for trajectory tracking.  The kitbot runs at ~10 :math:`ft/sec` which is ~3 :math:`m/sec`.
 
-**Max Acceleration:** The max acceleration of the robot for trajectory tracking. This does not need to be the maximum attainable acceleration of the robot, but just the max acceleration that you want to run trajectories at.
+**Max Acceleration:** The max acceleration of the robot for trajectory tracking. Using the same value for acceleration (3 :math:`m/sec^2`) is a good place to start depending if you don't know your drivetains characteristics.
```

Comment:
Probably want ``drivetrain's``, rather than ``drivetains`` here.  May read more smoothly if you remove 'depending' , as well.

The 'right' numbers here are pretty dependent on total robot weight, height, and center of gravity, and I can definitely say that they're a little too high for taller robots.  The Trajectory Drive tutorial uses the same values, though, without much caveat, so they may just be something the frc-docs group is comfortable with.

Reply:
We are the frc-docs group and no we don't have a certain "comfort" with these values we just want to give people a number to start with.  We can't make a number that works for every robot.

Thanks for the grammar suggestions I agree.